code/src/Components/ThoughtsMessage.js

```diff
@@ -0,0 +1,32 @@
+import React from 'react'
+import moment from 'moment'
+
+import './ThoughtsMessage.css'
+
+export const ThoughtsMessage = ({messageDetails, onHeartsChange}) => {
+
+    const handleClick = () => {
+        onHeartsChange(messageDetails._id);
+    };
+
+   return (
+    <article className="message-container">
+        <h3 className="message-text">{messageDetails.message}</h3>
+        <span className="form-message-footer">
+            <p>
+                <button id="hearts-button"
```

Comment:
I don´t think we are suppose to use "id".

Reply:
@nathalienordstrom I used id instead of class name as the css wouldn't implement on the button when using the class name for some strange reason. Worked around it by using id and it worked 😊

Reply:
Hmm, weird! It's acceptable as a workaround though